examples/glusterfs/README.md

```diff
@@ -59,6 +59,8 @@ Here is a snippet of [glusterfs-endpoints.json](glusterfs-endpoints.json),
 
 ```
 
+We should also create a service for this endpoints, so that it will be persistented. You can see [glusterfs-service.json](glusterfs-service.json) for details.
```

Comment:
Please add the command line that one would use to do this (I know it's simple but we try to make things as easy as possible for the reader).

Please also explain that this service is unique in lacking a selector, and that the reason for that is that you'll add the endpoints manually.